example/rpc/consumer_handler.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const {Producer} = require('../../index');
+const config = require('./config.json');
+
+module.exports = async function (payload, job_info) {
+	console.log('\tpayload:', payload, '\n\tjob info:', job_info);
+	console.log('JOB is started');
+	console.log('Replay TubeID ' + payload.replayTubeId);
```

Comment:
Should it be named "Reply Tube" rather than "Replay Tube"? I think the name with "Reply" makes more sence 😂

Reply:
Yes it makes more sence 😂